.github/abstract/Creating Workflows.md

```diff
@@ -0,0 +1,12 @@
+### What is a workflow?
+A `Workflow` is a description for a sequence of `FlowRepresentable`. Those operations are normally displaying screens or views. There are different ways of [creating workflows in SwiftUI](Creating%20Workflows%20in%20SwiftUI.html) and [creating workflows in UIKit](Creating%20Workflows%20in%20UIKit.html). 
+
+### What is a `FlowRepresentable`?
+A `FlowRepresentable` is a protocol that describes a type that can appear in a `Workflow`. It optionally declares `FlowRepresentable.WorkflowInput` and `FlowRepresentable.WorkflowOutput` indicating whether it requires data or whether it passes data forward. It also provides an initializer so that it can be created by the `Workflow`. Once an item conforms to `FlowRepresentable` it is capable of being injected into any `Workflow`. 
+
+### Valid workflow sequences
+Workflows enforce (either at compile-time or run-time) that the sequence of `FlowRepresentables` are well-formed. For example if a particular `FlowRepresentable` has a `FlowRepresentable.WorkflowInput` of `String` but the previous item passes an `Int` forward, that workflow is malformed. It's possible for a `FlowRepresentable` to declare a `FlowRepresentable.WorkflowInput` of `Never` if it does not take in data, `Any` if it can intake multiple types, or `AnyWorkflow.PassedArgs` if it can handle being passed no data, or data of any kind. Lastly, if your `FlowRepresentable.WorkflowInput` needs to take in multiple values you can use a tuple.
```

Comment:
More Grammarly-ed changes

```suggestion
Workflows enforce (either at compile-time or run-time) that the sequence of `FlowRepresentables` is well-formed. For example, if a particular `FlowRepresentable` has a `FlowRepresentable.WorkflowInput` of `String`, but the previous item passes an `Int` forward, that workflow is malformed. A `FlowRepresentable` can declare a `FlowRepresentable.WorkflowInput` of `Never` if it does not take in data, `Any` if it can intake multiple types, or `AnyWorkflow.PassedArgs` if it can handle being passed no data, or data of any kind. Lastly, if your `FlowRepresentable.WorkflowInput` needs to take in multiple values, you can use a tuple.
```